genomics-geek/dgd-wdl-workflows
# Create & split sample id check

Dear Maintainers,

Please accept this PR that addresses the following issues:
+[ #153 in clin-air-wdl](https://github.research.chop.edu/DGD/clin-air-wdl/issues/153)

Testing Done:
+ passes WOMtool ✅ 

TODO:
- check to see if this is appropriate 
-- should this be split up?
-- if so, should the workflow be under `workflows/`?
- change `import` statement in workflow to get new task
